iphone/Classes/TiUITextField.m

```diff
@@ -305,6 +305,13 @@ - (void) dealloc
 		NSNotificationCenter * theNC = [NSNotificationCenter defaultCenter];
 		[theNC addObserver:self selector:@selector(textFieldDidChange:) name:UITextFieldTextDidChangeNotification object:textWidgetView];
 	}
+  
+    if (![TiUtils isIOS10OrGreater]) {
+        NSString *str = ((UITextField *)textWidgetView).text;
+        ((UITextField *)textWidgetView).text = @"";
+        ((UITextField *)textWidgetView).text = str;
+    }
```

Comment:
Please update the code to the snippet we talked about today and migrate the indentation to tabs in this example, because it uses legacy-code. For new classes, we always use spaces, but in this case it's for consistency.